lib/pbxProject.js

```diff
@@ -1739,8 +1774,7 @@ pbxProject.prototype.getFirstProject = function() {
 }
 
 pbxProject.prototype.getFirstTarget = function() {
-
-    // Get first targets UUID
+    // Get first target's UUID
```

Comment:
I raised PR #68 to keep this fix in its own commit. If we merge PR #68, this fix should drop out if we do a squash merge.

Reply:
Undid the change in https://github.com/apache/cordova-node-xcode/pull/56/commits/a1b0da2c7017c3e8a3d0590c0ed22f26c26a6d88. I guess I'm too used to being liberal in my random 'cleanups'. :) Will keep them in separate commits/PRs for these repos in the future.